### PR TITLE
Remove nests for good and remove one button

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Remove nest being buildable for good and merge select resin and build resin in one button

## Why It's Good For The Game

Buttons bloat begone

## Changelog
:cl:
fix: Nests are really gone now
qol: Select resin and build resin buttons are merged
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 1]
```diff
@@ -183,47 +183,44 @@
...
+	name = "Secrete Resin"
 	action_icon_state = "resin wall"
-	mechanics_text = "Selects which structure you will build with the (secrete resin) ability."
-	keybind_signal = COMSIG_XENOABILITY_CHOOSE_RESIN
+	mechanics_text = "Builds whatever resin you selected"
+	ability_name = "secrete resin"
+	plasma_cost = 75
+	keybind_signal = COMSIG_XENOABILITY_SECRETE_RESIN
+	var/base_wait = 1 SECONDS
+	var/scaling_wait = 1 SECONDS
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:195]: Autodoc

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/abilities.dm (hunk 1 above)
Comment (BraveMole) [reply code/modules/mob/living/carbon/xenomorph/abilities.dm:195]: no clue what it does

Comment (SplinterGP) [reply code/modules/mob/living/carbon/xenomorph/abilities.dm:195]: depending of building thing it takes longer to build
```
/datum/action/xeno_action/activable/secrete_resin/proc/get_wait()
	. = base_wait
	if(!scaling_wait)
		return
	var/mob/living/carbon/xenomorph/X = owner
	var/sticky_resin_modifier = 1
	if(X.selected_resin == /obj/effect/alien/resin/sticky) //Sticky resin builds twice as fast
		sticky_resin_modifier = 0.5
	return (base_wait + scaling_wait - max(0, (scaling_wait * X.health / X.maxHealth))) * sticky_resin_modifier
```

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 2]
```diff
@@ -183,47 +183,44 @@
...
+/datum/action/xeno_action/activable/secrete_resin
+	name = "Secrete Resin"
 	action_icon_state = "resin wall"
-	mechanics_text = "Selects which structure you will build with the (secrete resin) ability."
-	keybind_signal = COMSIG_XENOABILITY_CHOOSE_RESIN
+	mechanics_text = "Builds whatever resin you selected"
+	ability_name = "secrete resin"
+	plasma_cost = 75
+	keybind_signal = COMSIG_XENOABILITY_SECRETE_RESIN
+	var/base_wait = 1 SECONDS
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:194]:
```suggestion
  ///Minimum time to build a resin structure
	var/base_wait = 1 SECONDS
```

File: code/modules/mob/living/carbon/xenomorph/abilities.dm (hunk 1 above)
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:195]:
```suggestion
 ///Multiplicator factor to add to the building time, depends on the health of the structure built
	var/scaling_wait = 1 SECONDS
```

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 3]
```diff
@@ -183,47 +183,46 @@
...
 	action_icon_state = "resin wall"
-	mechanics_text = "Selects which structure you will build with the (secrete resin) ability."
-	keybind_signal = COMSIG_XENOABILITY_CHOOSE_RESIN
+	mechanics_text = "Builds whatever resin you selected"
+	ability_name = "secrete resin"
+	plasma_cost = 75
+	keybind_signal = COMSIG_XENOABILITY_SECRETE_RESIN
+  ///Minimum time to build a resin structure
+	var/base_wait = 1 SECONDS
+ ///Multiplicator factor to add to the building time, depends on the health of the structure built
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:196]:
```suggestion
  ///Multiplicator factor to add to the building time, depends on the health of the structure built
```

Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm]:
```suggestion
  ///Multiplicator factor to add to the building time, depends on the health of the structure built
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**